src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -2477,11 +2477,13 @@
       },
       "PaymentMethodManagementType": {
         "type": "string",
-        "description": "Payment method management type",
+        "description": "Describes how to manage the payment method authorization flow in wallet and eCommerce domain:\n- REDIRECT if it must be managed with a redirect flow;\n- ONBOARDABLE if it must be managed with NPG and it is possible to save the payment method in the wallet, but also guest payment is accepted;\n- NOT_ONBOARDABLE if it must be managed with NPG but the method cannot be saved, only guest payment is accepted;\n- ONBOARDABLE_ONLY if it must be managed with NPG and it is mandatory to save the payment method in the wallet to use it. Guest payment isn't accepted;\n- ONBORDABLE_WITH_PAYMENT if it must be managed with NPG and it is possible to save it, to use it as guest payment, and to onboard it during the payment;",
         "enum": [
           "ONBOARDABLE",
           "NOT_ONBOARDABLE",
-          "REDIRECT"
+          "REDIRECT",
+          "ONBOARDABLE_ONLY",
```

Comment:
I think it's better to delete these two categories for checkout, currently checkout doesn't include `ONBOARDABLE_ONLY` and `ONBOARDABLE_WITH_PAYMENT`. What do you think?

Reply:
To be completely honest, I think that for all payment methods document the `onboardable` set has the same meaning, that is to be paid by NPG flux. Checkout doesn't care if a method is available for onboarding or not, or if it allows to make a contextual onboarding and payment, or if it is available for guest payment in app IO. 

If we should mantain a semantic value to all the string for this enum, we could at this point add also another value in payment-methods list, that is `NPG` and make 2 different list values for checkout and appIO/wallet. The first that has only `REDIRECT` and `NPG` and the second that contains all values except `NPG`.

Reply:
ok, I agree for now :-)